client/layout/activity-panel/activity-card/README.md

```diff
@@ -33,3 +33,31 @@ render: function() {
 * `date`: The timestamp associated with this activity.
 * `icon`: An icon or avatar used to identify this activity. Defaults to Gridicon "notice-outline".
 * `unread`: If this prop is present, the card has a small red bubble indicating an "unread" item. Defaults to false.
+
+ActivityCardPlaceholder
+=======================
+
+This component is similar to `ActivityCard` in output, but renders no real content, just loading placeholders. This is also hidden from any interaction with screen readers using `aria-hidden`.
+
+It does expect `children`, which should be spans with class `is-placeholder`. This will let you define how many lines of placeholder content you want to show.
```

Comment:
Instead of requiring a number of spans to be set as `children`, maybe we could make this a bit easier and just have a prop for number of lines? Then that number of spans could be output in the component.

`<ActivityCardPlaceholder hasDate numberOfLines={ 1 } />`

Reply:
I like the `numberOfLines` approach too.